server/repository/src/db_diesel/stock_line.rs

```diff
@@ -188,7 +201,7 @@ fn create_filtered_query(filter: Option<StockLineFilter>) -> BoxedStockLineQuery
         } = f;
 
         apply_equal_filter!(query, id, stock_line_dsl::id);
-        apply_equal_filter!(query, item_id, stock_line_dsl::item_id);
+        apply_equal_filter!(query, item_id, item::id);
```

Comment:
I'm not sure if this change is good haha. The alternative is `item_link::item_id` I think. Rust compiler should apply to my comments it'd help me be more confident of that statement.